src/mono/mono/component/debugger-agent.c

```diff
@@ -7940,6 +7950,42 @@ domain_commands (int command, guint8 *p, guint8 *end, Buffer *buf)
 		buffer_add_objid (buf, o);
 		break;
 	}
+	case MDBGPROT_CMD_APPDOMAIN_GET_TYPE: {
```

Comment:
it it only works on arrays? why is it called "get type" and not something like "create array type"

Reply:
It works also for pointers, but I'm not sure how to implement it yet, that is why I didn't implement.